.projenrc.ts

```diff
@@ -80,14 +81,14 @@ const project = new awscdk.AwsCdkConstructLibrary({
   npmAccess: NpmAccess.PUBLIC,
 
   publishToPypi: {
-    distName: PUBLICATION_NAMESPACE+'.'+PROJECT_NAME,
-    module: (PUBLICATION_NAMESPACE.replace(/-/g, '_'))+'.'+(PROJECT_NAME.replace(/-/g, '_')), // PEP 8, convert hypens
+    distName: PUBLICATION_NAMESPACE + '.' + PROJECT_NAME,
+    module: (PUBLICATION_NAMESPACE.replace(/-/g, '_')) + '.' + (PROJECT_NAME.replace(/-/g, '_')), // PEP 8, convert hypens
```

Comment:
Adding the ESlint configuration around line 215 will fix future format spacing like here

```
project.eslint?.addRules({ 'space-infix-ops': ['error', { int32Hint: false }] });
```